.github/workflows/setup_env.sh

```diff
@@ -32,21 +32,38 @@ export top=$(pwd)
 
 shopt -s expand_aliases
 
-
-print "======================================== Load compiler"
-quiet module load gcc@7.3.0
-quiet which g++
-g++ --version
+quiet module load python
+quiet which python
+quiet which python3
+python  --version
+python3 --version
 
 quiet module load pkgconf
 quiet which pkg-config
 
+export color=no
+export CXXFLAGS="-Werror -Wno-unused-command-line-argument"
+
+#----------------------------------------------------------------- Compiler
+if [ "${compiler}" = "intel" ]; then
+    print "======================================== Load Intel oneAPI compiler"
+    quiet module load intel-oneapi-compilers
+else
+    print "======================================== Load GNU compiler"
+    quiet module load gcc@8.5.0
```

Comment:
Is it normal to have this module name `gcc@8.5.0`, especially with the symbol @?
Also, does it have to be consistent with all the machines?

Reply:
Yes, it's normal: https://modules.readthedocs.io/en/stable/modulefile.html#version-specifiers
All machines load the same Docker container, so they have the same versions available.